merge_sort.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/local/bin/python3
+
+import argparse
+
+from util.sort_util import SortableCollection
+
+def get_sortable_collection():
+    return SortableCollection([5, 1, 4, 2, 8])
+
+def process_args():
+    pass
+
+def sort(collection: list) -> list:
+
+    def merge(left: list, right: list) -> list:
+
+        def _merge():
+            while left and right:
+                yield (left if left[0] <= right[0] else right).pop(0)
```

Comment:
Please use the swap and compare methods provided by SortableCollection. This will enable the algorithm to be compared meaningfully to other algorithms.